docker/dockerfiles/allin1/services/director/run.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
```

Comment:
Better to add license header for these new added files.

Reply:
done